<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TODO_LIST</class>
 <widget class="QMainWindow" name="TODO_LIST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_todo">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>추가할 내용을 입력하세요</string>
    </property>
   </widget>
   <widget class="QPushButton" name="out_todo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>파일로 저장하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="all_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>671</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
